src/components/BookList.js

```diff
@@ -17,22 +20,41 @@ const BookList = ({ books }) => {
   };
 
   return (
-    <ul>
+    <div className="book-list">
       {books.map((book) => (
-        <li key={book.itemId || uuidv4()}>
-          <h3>{book.title}</h3>
-          <p>
-            Author:
-            {book.author}
-          </p>
-          <p>
-            Category:
-            {book.category}
-          </p>
-          <button type="submit" onClick={() => handleRemoveBook(book.itemId)}>Remove Book</button>
-        </li>
+        <div key={book.itemId || uuidv4()} className="book-details">
+          <div className="book-info">
+            <p className="School-of Text-Style-9">
+              {book.category}
+            </p>
+            <h3 className="Title Text-Style-5">{book.title}</h3>
+            <div className="author-buttons">
+              <p className="Suzanne-Collins Text-Style-8">
+                {book.author}
+              </p>
+              <ButtonGroup variant="text" aria-label="text button group">
+                <Button type="submit">Comments</Button>
+                <Button type="submit" onClick={() => handleRemoveBook(book.itemId)}>Remove</Button>
+                <Button type="submit">Edit</Button>
+              </ButtonGroup>
+            </div>
+          </div>
+          <div className="progress">
+            <CircularProgress variant="determinate" value={75} />
+            <p>
+              75%
+              <br />
+              Completed
+            </p>
```

Comment:
- [ ] I recommend giving the number the right size and font weight since your design is all the same but it should have more font weight and size so please make them like the given design, it is very important to make our design full of life to be attractive to any user to see it 😊⭐✨ especially recruiters in the future 💯🙏🏻


- Your
![image](https://github.com/Eccb7/book-store/assets/58553711/9ad6dd99-f66c-44bb-856b-5c0f8c21a503)

- Given
![image](https://github.com/Eccb7/book-store/assets/58553711/ac363c69-c832-44ea-a308-443f69298c38)